cedar-policy/src/api.rs

```diff
@@ -694,13 +694,13 @@ pub enum SchemaError {
     #[error("{0} is not a record")]
     ContextOrShapeNotRecord(ContextOrShape),
     /// An Action Entity (transitively) has an attribute that is an empty set
-    #[error("Action attribute is an empty set")]
-    ActionEntityAttributeEmptySet,
+    #[error("Action `{0}` attribute is an empty set")]
+    ActionEntityAttributeEmptySet(EntityUid),
```

Comment:
These changes to public `SchemaError` enum are breaking, it might be helpful to separate them out from the other changes so that we can get the improved error messages in the release branch.  (Or maybe the PR to the release branch could just undo the structural enum changes here)

Reply:
My preferred solution would be splitting them to a separate commit within this PR so that the PR to the release branch can include just that commit.